### ISSUE TITLE
Release 2.71

### ISSUE THREAD

**Issue by gamontoya** [ucsdlib/damsmanager#314]
### Descriptive summary

Version: 2.71
Release Manager: @lsitu
Product Owner and Reviewers: @gamontoya
Operations: @jhriv or @rstanonik 

-----

### To Verify

- [x] #307 - Add Person:Advisor to Excel Standard input stream
- [ ] #301 - Support batch export metadata in RDF and CSV format
- [ ] #284 - RDCP download counts are inaccurately reported

**Comment by lsitu** [comment]
@mcritchlow I've added PR https://github.com/ucsdlib/damsmanager/pull/315 to disable the automatic process for CIL Ingest for now since the feature is still under review for CIL API implementation. I think we can move all other features to staging and prod once the PR is merged. Thanks.

**Comment by lsitu** [comment]
@gamontoya / @hjsyoo / @GregReser DAMS Manager release 2.71 are ready on staging for test / review now. Thanks.

**Comment by gamontoya** [comment]
@GregReser Testing for:

- Add Person:Advisor to Excel Standard input stream
- Support batch export metadata in RDF and CSV format

@hjsyoo Testing for:
- RDCP download counts are inaccurately reported

**Comment by GregReser** [comment]
@lsitu @gamontoya Confirming that Add Person:Advisor to Excel Standard input stream is working

**Comment by GregReser** [comment]
@lsitu I have feedback on Support batch export metadata in RDF and CSV format.  Should I post that here?

**Comment by lsitu** [comment]
@GregReser If it's something need to adjust, posting here is fine. But feel free to create a new ticket for new features or reopen the original ticket https://github.com/ucsdlib/damsmanager/issues/301 for bugs. Thanks.

**Comment by GregReser** [comment]
@lsitu I think it's a bug, so I'll reopen #301

**Comment by lsitu** [comment]
@GregReser I've deployed damsmanager with a hotfix for the issues you found to staging. Could you test it on staging again? Thanks.

**Comment by GregReser** [comment]
@lsitu The issues are fixed and everything is working.  This is ready to deploy to production.  
This is a huge benefit to us! Thank you very much for your work on it.

**Comment by lsitu** [comment]
Thanks @GregReser. 
@gamontoya Are we ready to move damsmanager release 2.71 to prod?

**Comment by hjsyoo** [comment]
@gamontoya @lsitu I won't be able to properly test the RDCP download count fix without holding this release up. If it's fine for you, I'm all for going ahead with the release. I can test later and create a new ticket if there are any issues with this release. Sound ok?

**Comment by lsitu** [comment]
@hjsyoo I think that works since it's not easy to test dams statistics on staging. @gamontoya What do you think?

**Comment by gamontoya** [comment]
@lsitu Sounds good.

**Comment by lsitu** [comment]
@rstanonik / @jhriv  The codes for damsmanager release 2.71 is already in the master branch and it's ready to be deployed to prod now. Thanks you.

**Comment by rstanonik** [comment]
deployed to prod

**Comment by GregReser** [comment]
Confirmed that Add Person:Advisor to Excel Standard input stream is working properly

**Comment by GregReser** [comment]
Confirmed that batch export metadata in RDF and CSV format is working properly.

**Comment by gamontoya** [comment]
Kamino cloned this issue to [ucsdlib/damsmanager](https://github.com/ucsdlib/damsmanager/issues/339)
